airflow/stats.py

```diff
@@ -342,7 +343,7 @@ def timer(self, stat=None, *args, tags=None, **kwargs):
 
 
 class _Stats(type):
-    factory = None
+    factory: Callable[[], StatsLogger]
```

Comment:
By not setting this to `None` we have easier typchecking of calling `cls.__class__.factory`